### PR TITLE
http-worker: refactor & fix curl_easy_getinfo error message

### PR DESCRIPTION
**Pull request by lbudai**
result of curl_easy_getinfo was not used (CURLcode code), the result of
the previus curl_xxx call was used (CURLcode ret)

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE

File: modules/http/http-worker.c [hunk 1]
```diff
@@ -289,61 +289,91 @@ _finish_request_body(HTTPDestinationWorker *self)
     g_string_append_len(self->request_body, owner->body_suffix->str, owner->body_suffix->len);
 }
 
-static LogThreadedResult
-_flush_on_target(HTTPDestinationWorker *self, HTTPLoadBalancerTarget *target)
+static void
+_debug_flush_on_target(HTTPDestinationWorker *self, HTTPLoadBalancerTarget *target, glong http_code)
```
Comment (furiel) [inline modules/http/http-worker.c]: May I suggest calling `_debug_response_info`. The current name is a little confusing as no flush is happening here.

File: modules/http/http-worker.c [hunk 2]
```diff
@@ -289,61 +289,91 @@ _finish_request_body(HTTPDestinationWorker *self)
...
     }
 
-  glong http_code = 0;
+  return TRUE;
+}
 
-  CURLcode code = curl_easy_getinfo(self->curl, CURLINFO_RESPONSE_CODE, &http_code);
-  if (code != CURLE_OK)
+static gboolean
+_curl_get_info(HTTPDestinationWorker *self, HTTPLoadBalancerTarget *target, glong *http_code)
```
Comment (furiel) [inline modules/http/http-worker.c]: Can you rename this `_curl_get_status_code`?

Comment (lbudai) [reply modules/http/http-worker.c]: yes, that sounds better. thanks!

**Comment by kira-syslogng** [comment]
Build FAILURE

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**